sfcasts/ep1/computed.md

```diff
@@ -1,228 +1,192 @@
 
 Whenever you need to calculate a value, like an array of classes, based off some
-`props` or `data`, the way to do that is with a `computed property`. Here's how it
-works: Down inside of our JavaScript code, right below `data`, we're going to add a
-new property called `computed` that is going to be set to an object.
-
-
-Just like `methods` down here, we will populate this with a series of functions.
-So let's add our first computed property here called `componentClass()`. You can
-call that *anything*. This is going to return the array of classes that this component
-should have at any given time. Don't worry about how we're going to use it yet. Let's
+`props` or `data`, the way to do that is with a *computed property*. Here's how it
+works: Down inside our JavaScript code, right below `data`, add a
+new option called `computed` and set it to an object.
+
+Just like `methods` down here, we'll populate this with functions.
+Add our first computed property called `componentClass()`. You can
+name that *anything*. This will return the array of classes that this component
+should have. Don't worry about how we're going to *use* it yet: let's
 just start by filling in the logic. First we'll say `const classes =` and set that
 to an array with the three classes that we always need. `this.$style.component` (we'll
 talk about that in a second), `p-3`, and `mb-5`. Perfect!
- 
 
-At the bottom we'll return `classes`. We'll worry about that conditional class
-in a second. 
- 
-## .this magic 
+At the bottom, return `classes`. We'll worry about that conditional class
+in a second.
 
-Let's talk about `$this.style.component` real quick:
+## .this magic
 
-We know that as soon as we add `module` to our style tag, `Vue` makes a new `$style` 
-object available in our template. We use that to say things like `$style.component` 
-where it resolves to the actual name of the class once it gets compiled by webpack.
+But let's talk about `$this.style.component` real quick:
 
-A few minutes ago, we learned that anytime you referenced a variable inside of
-a template internally, what that actually does is it calls `this.$style`, where
-`$style` is a property on the `vue` instance.
+We know that as soon as we add `module` to our style tag, `Vue` makes a new `$style`
+object available in our template. We use that to say things like `$style.component`.
 
-***TIP
-Notice that the template also understands the `this` keyword, be we omit it
-as a shortcut.
-***
+A few minutes ago, we learned that anytime you reference a variable inside of
+a template, internally, what that actually does is call `this.$style`. We don't
+*have* to say `this.` in the template because Vue adds this for us automatically.
 
 So even if we knew nothing else, the very fact that we can reference the `$style`
-object in a template means that the `vue` instance has a `$style` property in it.
+variable in a template means that the `vue` instance has a `$style` property in it.
```

Comment:
It's an object, not a variable. Style cannot be `"a string"` or `true`.

Reply:
Definitely. But the key thing we're showing here is that if you can reference "any variable" in a template, it means that this thing must be a property. The fact that `$style` is an object isn't the main relevant piece that I want them to focus on. Does that make sense?

Reply:
Ok! No big deal